crates/duty-tracker/src/contract_state_machine.rs

```diff
@@ -335,6 +362,9 @@ pub enum ContractState {
     /// This state describes everything from the moment the post-assert transaction confirms, to
     /// the moment either the disprove transaction confirms or the payout transaction confirms.
     Asserted {
+        /// The txid of the deposit request transaction that kicked off this contract.
+        deposit_request_txid: Txid,
+
```

Comment:
Why do we still have these?

Reply:
Removed in f6255bbbdbb215dbc182efa2df0f43d0821067d0